pandas/tests/arrays/categorical/test_operators.py

```diff
@@ -186,6 +186,19 @@ def test_comparison_with_unknown_scalars(self):
         tm.assert_numpy_array_equal(cat != 4,
                                     np.array([True, True, True]))
 
+    def test_comparison_with_known_scalars(self):
```

Comment:
can you rename this test to something like `test_compatison_with_missing_values` ?

Reply:
Will do!